### PR TITLE
B-21365 Customer UB Shipment Card MAIN

### PR DESCRIPTION
**Pull request by traskowskycaci**
This branch contains 21362/21363. Do not merge into main until 21362/21363's PR is in main: https://github.com/transcom/mymove/pull/13985

## [INT PR](https://github.com/transcom/mymove/pull/13878)

"[fix test](https://github.com/transcom/mymove/pull/13986/commits/28b92e52f7fbe01cb72ac48b2e9fae3f83b62309)" commit comes from the last commit of this int PR:
https://github.com/transcom/mymove/pull/13998/commits

The other two commits in [that int PR](https://github.com/transcom/mymove/pull/13998/commits) are already in main with this PR: 
https://github.com/transcom/mymove/pull/13992

### CODE REVIEW

**Review by cameroncaci — COMMENTED** [review]
Diff is the cause of cherry-pick SHA changes. Good to approve once the upstream is through

**Comment by traskowskycaci** [comment]
> Diff is the cause of cherry-pick SHA changes. Good to approve once the upstream is through

Yes, flow of work for this since I relied on int only work was make the change in int, cherry-pick to main, then merge main branch into int branch. Main branch has duplicated commits, one each for the cherry pick and then one that matches int where the SHA should match int commit

File: .circleci/config.yml [hunk 1]
```diff
@@ -877,8 +877,8 @@ commands:
             export FEATURE_FLAG_MANAGE_SUPPORTING_DOCS=false
             export FEATURE_FLAG_THIRD_ADDRESS_AVAILABLE=false
             export FEATURE_FLAG_QUEUE_MANAGEMENT=false
-            export FEATURE_FLAG_ENABLE_ALASKA=false
             export FEATURE_FLAG_UNACCOMPANIED_BAGGAGE=false
+            export FEATURE_FLAG_ENABLE_ALASKA=false
```
Comment (danieljordan-caci) [inline .circleci/config.yml:881]: This seems to be the only change - any idea why it got moved here but not INT?

Comment (traskowskycaci) [reply .circleci/config.yml:881]: Likely merging in main - not sure

Comment (traskowskycaci) [reply .circleci/config.yml:881]: Matches current int: https://github.com/transcom/mymove/blob/integrationTesting/.circleci/config.yml
image

**Review by deandreJones — APPROVED** [review]

**Review by cameroncaci — APPROVED** [review]

**Pull request merged**